src/App/Pages/Send/SendAddEditPage.xaml

```diff
@@ -252,28 +252,33 @@
                         </StackLayout>
                         <StackLayout
                             Orientation="Horizontal"
-                            Spacing="0">
+                            Spacing="0"
+                            AutomationProperties.IsInAccessibleTree="True"
+                            AutomationProperties.Name="{Binding OptionsAccessilibityText}">
+                            <StackLayout.GestureRecognizers>
+                                <TapGestureRecognizer Tapped="ToggleOptions_Clicked"/>
+                            </StackLayout.GestureRecognizers>
                             <Button
                                 Text="{u:I18n Options}"
                                 x:Name="_btnOptions"
                                 StyleClass="box-row-button"
                                 TextColor="{DynamicResource PrimaryColor}"
                                 Margin="0"
-                                Clicked="ToggleOptions_Clicked"/>
+                                AutomationProperties.IsInAccessibleTree="False"/>
```

Comment:
Isn't the default value of `AutomationProperties.IsInAccessibleTree ` `False`? Is there a bug or something that this line is needed?

Reply:
Default value is Null. 
<img width="457" alt="image" src="https://user-images.githubusercontent.com/4648522/170495108-f84249bb-cc0a-4651-b33a-0acd90d85f0d.png">

Reply:
Oh ok great then 😄